modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -197,11 +375,11 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 The {productname} 6.0 release includes accompanying changes affecting the {productname} *self-hosted* services for the following plugins:
 
-* The Enhanced Media Embed plugin (`+mediaembed+`)
-* The Export plugin (`+export+`)
-* The Enhanced Image Editing plugin (`+editimage+`)
-* The Link Checker plugin (`+linkchecker+`)
-* The Spell Checker Pro plugin (`+tinymcespellchecker+`)
+* The xref:introduction-to-mediaembed.adoc[Enhanced Media Embed] plugin (`+mediaembed+`)
+* The xref:export.adoc[Export] plugin (`+export+`)
+* The xref:editimage.adoc[Enhanced Image Editing] plugin (`+editimage+`)
+* The xref:linkchecker.adoc[Link Checker] plugin (`+linkchecker+`)
+* The xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro] plugin (`+tinymcespellchecker+`)
```

Comment:
This shouldn't have been changed, as the links are just below this instead. As such it's quite odd to have them linked twice.

If we do want this changed though, then the template in https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/_new_content_templates/releasenotes_blank/x_y_z-release-notes-premium-changes.adoc also needs changing.

Reply:
Done - removed "The" and template file was also updated.